fix: OPTIC-411: [lsp] Fix Follow Redirects improperly handles URLs in the url.parse() function

robot-ci-heartex opened the pull request:
Hi @yyassi-heartex!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/7748626614) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/dm2/pull/299
- https://github.com/HumanSignal/label-studio-frontend/pull/1687

yyassi-heartex committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7748626614

yyassi-heartex committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7748647028

yyassi-heartex committed:
Merge branch 'develop' into 'fb-optic-411/dependency-update'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7818355749

yyassi-heartex committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7818355749

yyassi-heartex committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7818370566

yyassi-heartex committed:
Merge branch 'develop' into 'fb-optic-411/dependency-update'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7845481657

yyassi-heartex committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7845481657

yyassi-heartex committed:
Merge branch 'develop' into 'fb-optic-411/dependency-update'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7846202339

yyassi-heartex committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7846202339

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7846231428

hlomzik commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/7874809536)
> Successfully merged:  16 files changed, 65 insertions(+), 35 deletions(-)

hlomzik committed:
Merge branch 'develop' into 'fb-optic-411/dependency-update'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7874809536

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7874832479

yyassi-heartex committed:
Merge branch 'develop' into 'fb-optic-411/dependency-update'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7875865229

yyassi-heartex committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7875865229

yyassi-heartex committed:
[submodules] Copy src HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7877021932

yyassi-heartex committed:
[submodules] Copy src HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7877024862

robot-ci-heartex committed:
ci: Build frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/7877055836

yyassi-heartex committed:
Merge branch 'develop' into fb-optic-411/dependency-update
